tests/integration-tests/configs/common/common.yaml

```diff
@@ -8,9 +7,9 @@ ad_integration:
         schedulers: ["slurm"]
         benchmarks:
           - mpi_variants: ["openmpi"]
-            num_instances: [100]
+            num_instances: [5]
             osu_benchmarks:
-              collective: ["osu_allreduce", "osu_alltoall"]
+              collective: ["osu_alltoall"]
```

Comment:
This does not really matter because the benchmark are not running. The benchmark code is dormant